README.md

```diff
@@ -1,4 +1,15 @@
-Here I am solving two issues 
+# Pattern library documentation
+
+This reopository is made for solving two issues in the dgl-104 pattern library public repository. The issues are 
 
 1. Add React Native implementation for Observer pattern #9
 2. Add Java implementation of Template pattern #64
+
+  ## 1. Add React Native implementation for Observer pattern #9 
+
+  The folder called journals is responsible for this issue. This folder is contain the codes of a simple react native app that use the observer pattern. To implement that you should be having node js installed and by running the comman npm start or expo start will run this app. This app is not detailed about the UI. But checking the code you will be able to see how observer pattern is implemented in react-native.
+
+## 2. Add Java implementation of Template pattern #64   
+
+The file CandyFactory.java is responsible for solving this issue. In that file you will be seeing how the template pattern is implemented in java. 
+
```

Comment:
These comments don't really belong in the main README.md file. You could include them in your RRJ so that I know what you've done.